### ISSUE TITLE
Exporting issues from multiple projects I own

### ISSUE THREAD

**Issue by beansrowning** [mattduck/gh2md#36]
This might be a silly thought, but I'm curious if there's a way to bulk-export all issues/PRs across all projects without explicitly naming them all. I figured it would be easy enough to enumerate them all and loop through each, but that would also require constant updating anytime I had a new repo.

Thanks!

**Comment by mattduck** [comment]
Yeah this will definitely be possible and is a good idea. I'll build in support for it sometime. Won't get a chance to start on it for at least a week but I'll leave an update here when I get to it.
